src/components/layout/Sidebar.vue

```diff
@@ -1,15 +1,20 @@
 <template>
-	<aside class="column is-one-third">
```

Comment:
I like having the additional semantics from the `aside` element, any reason why you removed it?

Reply:
Oh that was a mistake, that aside should have been moved to the AppContent component then I automatically wrote a div over there.

Reply:
I fixed this in https://github.com/wmde/fundraising-app-frontend/pull/206 because I didn't want to have to rebase.